README_CN.md

```diff
@@ -0,0 +1,87 @@
+# ReZygisk
+
+[English](https://github.com/PerformanC/ReZygisk/blob/main/README.md)
+
+ReZygisk 是 Zygisk 的另一个独立实现，从 ZygiskNext 分叉而来，为 KernelSU、Magisk 和 APatch 提供 Zygisk API 支持(目前处于测试阶段)。
+
+项目致力于使用 C 语言重写原本的 C++ 和 Rust 代码，以便更加现代、高效地实现 Zygisk API，并使用更宽松的开源许可证。
+
+> [!NOTE]
+> 此模块还在测试阶段，请仅安装正式版本的压缩包。
+>
+> 您可以从 [Actions](https://github.com/PerformanC/ReZygisk/actions) 页面下载自动构建包，但要注意自负风险。使用不稳定的版本时，设备可能会陷入启动循环(Bootloop)。
+
+## 为什么要ReZygisk？
+
+最新版本的 Zygisk Next 并不开源，仅其核心开发者有权查阅全部源代码。这不仅阻止了其他开发者贡献代码，还阻止了他们对项目代码进行审计。Zygisk Next 是一个以超级用户(root)权限运行的模块，可以访问整个系统，闭源后存在重大安全隐患。
```

Comment:
In the context, "他们" refers to "其他开发者".

> Not only does that limit **our** ability to contribute to the project, but also impossibilities the audit of the code

I translate "our" here as the "other developers" of Zygisk Next, which is reasonable, since PerformanC was not the only one tended to contribute to Zygisk Next.

Reply:
So the key is who are **we** here.

In the context, **we** are other devs of Zygisk Next affected by their decision to hide the source code.



In contrast,

> **We** understand they have their reasons to keep the code closed-source, but we believe the contrary.

As you can see, here "we" refers to PerformanC, as "we" believe close-source is bad and tended to publish a open-source fork.

Hopefully I've explained it clearly.